packages/scripts/src/sitemaps/runner.ts

```diff
@@ -0,0 +1,20 @@
+import { logger } from '@hicommonwealth/logging';
+import {
+  createAsyncWriterS3,
+  createDatabasePaginatorDefault,
+  createSitemapGenerator,
+} from '@hicommonwealth/sitemaps';
+
+const log = logger(__filename);
+
+async function doExec() {
+  const writer = createAsyncWriterS3();
+  const paginator = createDatabasePaginatorDefault();
+
+  await createSitemapGenerator(writer, [
+    paginator.threads,
+    paginator.profiles,
+  ]).exec();
+}
+
+doExec().catch((err) => log.fatal('Unable to process sitemaps: ', err));
```

Comment:
We'll want to have a `statsd` heartbeat increment here since this job will run periodically. We have 3 options:
- Add it now following pattern in https://github.com/hicommonwealth/commonwealth/pull/7494
- Assume this PR gets in first and we add the heartbeat in https://github.com/hicommonwealth/commonwealth/pull/7494
- Ignore it for now but create a quick-follow ticket to be linked here and completed after this PR is merged.

Reply:
Let's take this option:

> Ignore it for now but create a quick-follow ticket to be linked here and completed after this PR is merged.

This PR is growing and I'd like to get it merged.